components/org.wso2.carbon.identity.oidc.dcr/pom.xml

```diff
@@ -151,6 +151,7 @@
                             org.wso2.carbon.identity.application.common.model; version="${carbon.identity.framework.imp.pkg.version.range}",
                             org.wso2.carbon.identity.application.mgt;version="${carbon.identity.framework.imp.pkg.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.*;version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.core.util;version="${carbon.identity.framework.version}",
```

Comment:
Need to add the version range here.

Reply:
Fixed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1447/commits/6dedaee605279dad7e577ffbed37a916393a6bb8